app/models/organisation_type.rb

```diff
@@ -1,2 +1,5 @@
 class OrganisationType < ApplicationRecord
+  def api_json
```

Comment:
We could possibly create an organisation_type model spec and test api_json in it but not sure whether it's worth it.

Reply:
i just added it 😄 before seeing this 👍